lambda/data.tf

```diff
@@ -18,4 +18,24 @@ data "aws_ssm_parameter" "staging_account_number" {
 data "aws_ssm_parameter" "prod_account_number" {
   count = var.project == "tdr" && local.environment == "mgmt" ? 1 : 0
   name  = "/mgmt/prod_account"
-}
\ No newline at end of file
+}
+data "aws_ssm_parameter" "backend_checks_client_secret" {
+  count = var.project == "tdr" && var.lambda_file_format ? 1 : 0
+  name  = "/${local.environment}/keycloak/backend_checks_client/secret"
+}
+
+data "aws_vpc" "current" {
+  tags = {
+    Name = "tdr-vpc-${local.environment}"
```

Comment:
Is is ok with the hardcoded "tdr" in the terraform-modules repo? I know we are treating the lambdas slightly differently to other modules in the repo

Reply:
They are different but seeing as the variable is already there and being used, I may as well use it here.